java/test/jmri/util/JUnitUtil.java

```diff
@@ -180,6 +181,10 @@ public static void setUpLoggingAndCommonProperties() {
             Log4JUtil.initLogging(filename);
         }
 
+        if (SystemConsole.isCreated()) {
+            SystemConsole.getInstance().open();
```

Comment:
Would it be better to simply close the SystemConsole streams immediately and have tests that need those streams to be open open them just for those tests?